weave-js/src/components/Panel2/toWeaveType.tsx

```diff
@@ -0,0 +1,126 @@
+import {isFunctionType, Type} from '@wandb/weave/core';
```

Comment:
@nickpenaranda Danny finally pulled out toWeaveType here, a great move. Its just in a file called toWeaveType right now. What do you think?

Reply:
This is fine, but ideally we'd access these things through the WeaveInterface